source/docs/software/telemetry/writing-sendable-classes.rst

```diff
@@ -0,0 +1,71 @@
+Writing Your Own Sendable Classes
+=================================
+
+Since the ``Sendable`` interface only has one method, writing your own classes that implement ``Sendable`` (and thus automatically log values to and/or consume values from the dashboard) is extremely easy: just provide an implementation for the overrideable ``initSendable`` method, in which setters and getters for your class's fields are declaratively bound to key values (their display names on the dashboard).
+
+For example, here is the implementation of ``initSendable`` from WPILib's ``BangBangController``:
+
+.. tabs::
+
+    .. code-tab:: java
+
+        .. remoteliteralinclude:: https://raw.githubusercontent.com/wpilibsuite/allwpilib/main/wpimath/src/main/java/edu/wpi/first/math/controller/BangBangController.java
```

Comment:
These 2 RLI's didn't work.  I am not sure if the problem is the lack of linenos or what.